### DISCUSSION TITLE
Documentation on Authorization $config Variables

### DISCUSSION THREAD

**Discussion by derekritchison** [netsuitephp/netsuite-php#135] [Q&A]
Is there an example of what specifically we should be plugging into the $config variables for endpoint, host and account? Whatever I experiment with I end up with one of two errors; either a generic "Invalid login attempt" or:

`PHP Fatal error:  Uncaught SoapFault exception: [WSDL] SOAP-ERROR: Parsing WSDL: Couldn't load from 'https://XXXXXXX-sb1.netsuite.com/wsdl/v2022_1_0/netsuite.wsdl' : failed to load external entity "https://3489459-sb1.netsuite.com/wsdl/v2022_1_0/netsuite.wsdl"`

...where XXXXXXX is my unique sandbox account ID.

**Comment by MichaelBrenden** [comment]
This works for us

$config = [
	// required -------------------------------------
	'endpoint'			=> '2023_1',
	'host'				=> 'https://MY_CUSTOMER_NUMBER.suitetalk.api.netsuite.com',
	'account'			=> 'MY_CUSTOMER_NUMBER',
	'consumerKey'		=> 'get this from inside your netsuite',
	'consumerSecret'	=> 'get this from inside your netsuite',
	'token'				=> 'get this from inside your netsuite',
	'tokenSecret'		=> 'get this from inside your netsuite',
	// optional -------------------------------------
	'signatureAlgorithm'	=> 'sha256', // Defaults to 'sha256'
	'logging'			=> true,
	'log_path'			=> '/var/www/priv/netsuite/logs',
	'log_format'		=> 'netsuite-php-%date-%operation',
	'log_dateformat'	=> 'Ymd.His.u',
];
$nss = new NSS( $config );
